### ISSUE TITLE
redirect: server for images not available

### ISSUE THREAD

**Issue by lthueer** [liqd/adhocracy-plus#1655]
https://aplus-hafencity-dev.liqd.net/dashboard/projects/alle-module-testen/information/

![Bildschirmfoto 2022-01-17 um 14 44 47](https://user-images.githubusercontent.com/36957474/149779882-86323386-12a2-4b0f-ba63-56f32658568b.png)

Note: for non-redirects it works

**Comment by fuzzylogic2000** [comment]
In what role did you attempt to do that? Initiator or admin?

**Comment by lthueer** [comment]
@fuzzylogic2000 As admin!

**Comment by fuzzylogic2000** [comment]
It is also the same for stage. But I don't know if that's good because it is not the Django update or bad because I don't know how to fix it.
![Screenshot from 2022-01-17 16-44-35](https://user-images.githubusercontent.com/8178179/149800221-897d99a9-3ece-48c5-a915-2c6109702538.png)
How important is it? Can we fix it with the upcoming story concerning the ck editor uploading? @fredewe @lthueer

**Comment by lthueer** [comment]
@fuzzylogic2000 I think we should fix this but it doesnot have high prio. I also checked for some redirects on prod which is weird that they didn't realise so far. Apparently none of them uploaded images to the info and result tab on redirects... On the a+ sites it works then. Maybe they just used the dashboard adhocracy.plus domains then..
What do you think @fredewe ?

**Comment by fredewe** [comment]
@fuzzylogic2000 fine for me! Thanks!

**Comment by goapunk** [comment]
@fredewe @lthueer this should be fixed on aplus-stage/dev and diid-stage/dev, if you could test this we can deploy it on prod and close the issues

**Comment by lthueer** [comment]
@goapunk @fredewe adhocracy+ looks great on dev + stage for me. Can't test diid. Thank you!

**Comment by fredewe** [comment]
@fuzzylogic2000 @goapunk @lthueer i tested DIID and Dev (redirect) and found the issue, that it is not possible to browse the server. There is a 404 (see screenshot).
<img width="1025" alt="Bildschirmfoto 2022-04-06 um 14 55 20" src="https://user-images.githubusercontent.com/5470479/161979645-bec7ddf0-4b5e-43ee-8213-cb2c31167be8.png">

**Comment by goapunk** [comment]
@fredewe good catch! I fixed it on diid-dev, will do the same for stage and aplus. It's not unlikely that there are still other places which are broken but I guess we'll have to fix them as they come

**Comment by fredewe** [comment]
@goapunk Thanks! Looks good to me!
